.gulp/gulpfile.iced

```diff
@@ -21,14 +23,34 @@ task 'init', "" ,(done)->
   done()
 
 # Run language-specific tests:
-task 'test', 'type check generated code', [], (done) ->
+task 'test', '', ['test/typecheck', 'test/nodejs-unit', 'test/chrome-unit'], (done) ->
+  done();
+
+task 'test/typecheck', 'type check generated code', [], (done) ->
   await execute "tsc -p ./test/tsconfig.generated.json", defer _
   done();
 
-task 'test', 'run unit tests', [], (done) ->
+task 'test/nodejs-unit', 'run nodejs unit tests', [], (done) ->
   await execute "mocha", defer _
   done();
 
+task 'test/chrome-unit', 'run browser unit tests', [], (done) ->
+  count = 2;

```

Comment:
Yay for magic numbers! (Maybe a quick comment explaining this value might be a useful addition)

Reply:
Will update this to explain to readers what's going on